Subscribers/BaseSubscriber.php

```diff
@@ -112,20 +112,11 @@ public function registerMyTemplateDir()
     {
         $template = '';
         if ($this->Application()->Container()->has('shop')) {
-            if ($this->Application()
-                    ->Container()
-                    ->get('shop')
-                    ->getTemplate()
-                    ->getVersion() >= 3
-            ) {
-                $template = 'responsive';
-            } else {
-                $template = 'emotion';
-            }
+            $template = 'responsive';
         }
 
         $this->Application()->Template()->addTemplateDir(
-            $this->Path() . 'Views/' . $template, 'connect'
+            $this->Path() . "Views/$template/connect"
```

Comment:
If the shop is not available (see in the if-statement), it loads the backend views. I use `"View/$template/connect" instead of concatenating it. Imho it imrpoves the readability a bit.